src/replicache-options.ts

```diff
@@ -81,9 +81,26 @@ export interface ReplicacheOptions<MD extends MutatorDefs> {
    * Replicache will also log `'info'` and `'error'` messages. When set to
    * `'info'` we log `'info'` and `'error'` but not `'debug'`. When set to
    * `'error'` we only log `'error'` messages.
+   * Default is `'info'`.
    */
   logLevel?: LogLevel;
 
+  /**
+   * Enables custom handling of logs.
+   *
+   * By default logs are logged to the console.  If you would like logs to be
+   * sent elsewhere (e.g. to a cloud logging service like DataDog) you can
+   * provide an array of `LogSink`s.  Logs at or above
+   * [[ReplicacheOptions.logLevel]] are sent to each of these `LogSink`s.
+   * If you would still like logs to go to the console, include
+   * [[consoleLogSink]] in the array.
+   *
+   * ```ts
+   * logSinks: [consoleLogSink, myCloudLogSink],
```

Comment:
We should verify that the JSDocs for consoleLogSink is showing up in the generated docs

Reply:
good call it does not. I removed the linking.

Reply:
actually I was able to get the linking to work by changing the docusaurus-plugin-typedoc config to `excludeExternals: false`, which seems like the right thing since now it includes docs for `LogSink` and `consoleLogSink`.

Reply:
👍